src/teleop/gui/src/components/CommIndicator.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
Let's change the name of this to LEDIndictor or something of the sort, We don't use this for Comms anymore and currently we don't have any interfaces for comms.